Translations/zh_CN.po

```diff
@@ -6985,7 +6985,7 @@ msgstr "下​一​级: "
 #, fuzzy
 #| msgid " to Level {:d}"
 msgid "{:s} to Level {:d}"
-msgstr " 到 {:d} 级"
+msgstr "{:s} 到 {:d} 级"
```

Comment:
Don't forget to remove the fuzzy/needs review flag or it will not show up ingame.